pkg/test/framework/components/istio/operator.go

```diff
@@ -502,6 +573,7 @@ func createRemoteSecret(ctx resource.Context, cluster kube.Cluster) (string, err
 	cmd := []string{
 		"x", "create-remote-secret",
 		"--name", cluster.Name(),
+		"--namespace", "istio-system",
```

Comment:
We shouldn't be hard-coding the namespace ... we should get it from the config.

Reply:
good catch, fixed